be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -1086,8 +1098,14 @@ Status ArrayFileColumnIterator::next_batch(size_t* n, vectorized::MutableColumnP
         auto null_map_ptr =
                 static_cast<vectorized::ColumnNullable&>(*dst).get_null_map_column_ptr();
         size_t num_read = *n;
-        bool null_signs_has_null = false;
-        RETURN_IF_ERROR(_null_iterator->next_batch(&num_read, null_map_ptr, &null_signs_has_null));
+        if (_null_iterator) {
+            bool null_signs_has_null = false;
```

Comment:
add comnent when _null_iterator will be nullptr

Reply:
done!